pynest/examples/Potjans_2014/README.rst

```diff
@@ -54,6 +58,15 @@ The output will be saved in the ``data`` directory.
 
 The code can be `parallelized <https://nest-simulator.readthedocs.io/en/latest/guides/parallel_computing.html>`_ using OpenMP and MPI, if NEST has been built with these features.
 The number of threads (per MPI process) can be chosen by adjusting ``local_num_threads`` in ``sim_params.py``.
+
+
+.. important::
+
+   The microcircuit model does not run correctly on a single virtual process. This means
+   the number of threads must be greater than 1. You can either increase
+   the number of MPI processes or set ``local_num_threads`` to a value > 1.
```

Comment:
```suggestion
   the number of MPI processes or set either ``local_num_threads`` to a value > 1 or replace ``local_num_threads`` by ``total_num_virtual_procs`` and set it to a value > 1.
```
Not quite sure if this addition really is required, but it would be more complete.

Reply:
@ackurth Before we merge this, what is your opinion on this?